microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -38,227 +56,269 @@ To use experimental APIs, download a prerelease version of the WebView2 SDK from
+
+   ```reg
+   REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannels /v WebView2APISample.exe /t REG_SZ /d "0,1"
+   ```
 
-1. Copy the ADMX file into a Policy Definitions template folder, such as `C:\Windows\PolicyDefinitions`.
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.  Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine, and can result in unexpected behavior.
 
-1. Copy the ADML file into a matching locale folder within the `Policy Definitions` folder, such as a `C:\Windows\PolicyDefinitions\en-us`<!--keep /en-us here--> folder.
 
-1. Open the **Local Group Policy Editor**.  To do this, in the Windows search bar, type "group policy", and then select **Edit group policy**.
+##### [Environment variable](#tab/environment-variable)
 
-1. Expand **Local Computer Policy**, then **Computer Configuration** or **User Configuration**.  Then expand **Administrative Templates** > **Microsoft Edge WebView2**.
+Name: `WEBVIEW2_RELEASE_CHANNELS`  
+Value: `<string of desired channels (ex: 0,1)>`
 
-   ![Local Group Policy Editor dialog](./set-preview-channel-images/local-group-policy-editor.png)
+The environment variable is applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app.
 
-1. Select **Browser Executable Folder**.  The following screenshots apply to setting the **Browser Executable Folder**.  Alternatively, select **Release Channel Preference**, which uses similar dialogs.
 
-   ![Setting the Browser Executable Folder](./set-preview-channel-images/browser-executable-folder.png)
+##### [Group policy](#tab/group-policy)
 
-1. Select the **Show** button.
+Set the `ReleaseChannel` policy.
 
-1. Fill-in the **Show Contents** dialog.  In the **Value name** column, enter your app's `.exe` file name or Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
+* Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`
+* Value: `<string of desired channels (ex: 0,1)>`
 
-   ![The Show Contents dialog](./set-preview-channel-images/show-contents.png)
+Do either of the following:
+
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download). Refer to [Configure Microsoft Edge policy settings on Windows devices](/deployedge/configure-microsoft-edge) for more information.
+
+* Use the built-in policy on Intune. Refer to [Configure Microsoft Edge policy settings in Microsoft Intune](/mem/intune/configuration/administrative-templates-configure-edge) for more information.
+
+---
 
-1. Select **OK** to close the dialogs.
+<!-- ------------------------------ -->
+#### How to use an environment variable
```

Comment:
We should include an example of how to do this programmatically. The below manual instructions are fine to get a single user to change the order for themselves, but most devs would do this programmatically in their app process if this is the approach they decide to take.

Reply:
Rendered articles for review: 
* **Test upcoming APIs and features** (update to use new API)
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3053
   * [Public preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/prerelease-testing-self-hosting/microsoft-edge/webview2/how-to/set-preview-channel.md)
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel)
* **Prerelease testing** (new article)
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3053
   * [Public preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/prerelease-testing-self-hosting/microsoft-edge/webview2/how-to/prerelease-testing.md)
* **Self-hosting** (new article)
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting?branch=pr-en-us-3053
   * [Public preview](https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/prerelease-testing-self-hosting/microsoft-edge/webview2/how-to/self-hosting.md)

Reply:
Added